lib/stripe/stripe_client.rb

```diff
@@ -127,21 +136,21 @@ def self.should_retry?(error, method:, num_retries:)
       end
     end
 
-    def self.sleep_time(num_retries)
+    def self.sleep_time(num_retries, config: Stripe.configuration)
```

Comment:
I saw in the git history that these methods were intended to be functional. Didn't want to stray from that so opted for injecting the config.